packages/syntax-core/src/TextField/TextField.tsx

```diff
@@ -80,6 +81,10 @@ export default function TextField({
    * Value of the TextField
    */
   value: string;
+  /**
+   * Specified legal number intervals for an input field. Specifically for time or number. If for time specify for milliseconds. Must be a positive value.
```

Comment:
It needs to be a positive number but there is no clear way to enforce that in TS sadly so just point it out in the docs.
For reference: https://stackoverflow.com/questions/21224922/is-there-a-way-to-represent-a-non-negative-integer-in-typescript-so-that-the-com.